### PR TITLE
Add support for Kia Rio III (UB) and Hyundai TPMS sensors

### PR DESCRIPTION
**Pull request by rrrlasse**

### CODE REVIEW

File: src/devices/tpms_kia.c [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+    Copyright (C) 2022 Lasse Mikkel Reinhold, Todor Uzunov aka teou, TTiges, 2019 Andreas
+    Spiess, 2017 Christian W. Zuckschwerdt <zany@triq.net>
+
+    This program is free software; you can redistribute it and/or modify it under the terms of
+    the GNU General Public License as published by the Free Software Foundation; either version
+    2 of the License, or (at your option) any later version.
+*/
+
+/*
```
Comment (zuckschwerdt) [inline src/devices/tpms_kia.c:12]: should be `/**`
and remove 4 spaces for each line.

**Comment by rrrlasse** [comment]
Sorry, I had one more commit because I just saw that other devices tell users to use the "-s 1000000" option through the device name. Let me know if I need to do anything else.

**Comment by zuckschwerdt** [comment]
Thanks. I both instances "-s 1M"  would be preferred.

**Comment by rrrlasse** [comment]
Hmm please wait merging this. I think the pulse widths need to be 50 instead of 200. I missed that when you read a file (-r flag), it assumes 250k by default. And I recorded at 1M. Need a few days to sort it out :)

**Comment by zuckschwerdt** [comment]
If samples are tagged with e.g. ...`_1000k`... then reading will honor that sample rate.
https://triq.org/rtl_433/IQ_FORMATS.html#file-name-meta-data

**Pull request merged**